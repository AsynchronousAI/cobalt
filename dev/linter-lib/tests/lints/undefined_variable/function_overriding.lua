function global.name() end

function a.b() end
function a.c() end

function d:e() end
function d:f() end
