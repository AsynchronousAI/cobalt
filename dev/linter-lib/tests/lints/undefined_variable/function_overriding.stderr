error[undefined_variable]: `global` is not defined
  ┌─ function_overriding.lua:1:10
  │
1 │ function global.name() end
  │          ^^^^^^

error[undefined_variable]: `a` is not defined
  ┌─ function_overriding.lua:3:10
  │
3 │ function a.b() end
  │          ^

error[undefined_variable]: `a` is not defined
  ┌─ function_overriding.lua:4:10
  │
4 │ function a.c() end
  │          ^

error[undefined_variable]: `d` is not defined
  ┌─ function_overriding.lua:6:10
  │
6 │ function d:e() end
  │          ^

error[undefined_variable]: `d` is not defined
  ┌─ function_overriding.lua:7:10
  │
7 │ function d:f() end
  │          ^

